backend/substrapp/tasks/k8s_backend.py

```diff
@@ -24,7 +24,7 @@
 FS_GROUP = os.getenv('FS_GROUP')
 IMAGE_BUILDER = os.getenv('IMAGE_BUILDER')
 KANIKO_IMAGE = os.getenv('KANIKO_IMAGE')
-KANIKO_MIRROR = os.getenv('KANIKO_MIRROR')
+KANIKO_MIRROR = os.getenv('KANIKO_MIRROR').lower() == 'true'
```

Comment:
there is a to_bool helper in the settings common.py file that you can use to convert boolean config values into a python True/False value and then treat it as a bool everywhere